### ISSUE TITLE
Ability to add alt text to videos inserted

### ISSUE THREAD

**Issue by acnowland** [froala/wysiwyg-editor#4886]
There is already the option available for images that are inserted. We should have the same option to add alt text to any video that is inserted into froala as well. This falls well within the guidelines of making a site/tool ADA accessible as well. Utilizing Alt text + captions is good practice.
